norfair/drawing/draw_points.py

```diff
@@ -0,0 +1,307 @@
+        By default a sensible value is picked considering the frame size.
+    thickness : Optional[int], optional
+        Thickness or width of the line.
+    draw_labels : bool, optional
+        If set to True, the label is added to a title that is drawn on top of the box.
+        If an object doesn't have a label this parameter is ignored.
+    draw_ids : bool, optional
+        If set to True, the id is added to a title that is drawn on top of the box.
+        If an object doesn't have an id this parameter is ignored.
+    text_size : Optional[float], optional
+        Size of the title, the value is used as a multiplier of the base size of the font.
+        By default the size is scaled automatically based on the frame size.
+    text_color : Optional[int], optional
+        Color of the text. By default the same color as the box is used.
+    text_thickness : Optional[int], optional
+        Thickness of the font. By default it's scaled with the `text_size`.
+    hide_dead_points : bool, optional
+        By default the dead points of the TrackedObject are hidden.
+        A point is considered dead if the corresponding value of `TrackedObject.live_points` is set to False.
+        If all objects are dead the object is not drawn.
+        All points of a detection are considered to be alive.
+        Set this param to False to always draw all points.
+
+    Returns
+    -------
+    np.ndarray
+        The resulting frame.
+    """
+    #
+    # handle deprecated parameters
+    #
+    if color_by_label is not None:
+        warn_once(
+            'Parameter "color_by_label" on function draw_points is deprecated, set `color="by_label"` instead'
+        )
+        color = "by_label"
+    if detections is not None:
+        warn_once(
+            "Parameter 'detections' on function draw_points is deprecated, use 'drawables' instead"
+        )
+        drawables = detections
+    if label_size is not None:
+        warn_once(
+            "Parameter 'label_size' on function draw_points is deprecated, use 'text_size' instead"
+        )
+        text_size = label_size
+    # end
+
+    if drawables is None:
+        return
+    if color is None:
+        color = "by_id"
+    if thickness is None:
+        thickness = -1
+    if radius is None:
+        radius = int(round(max(max(frame.shape) * 0.002, 1)))
+
+    for o in drawables:
+        d = Drawable(o)
+
+        if hide_dead_points and not d.live_points.any():
+            continue
+
+        if color == "by_id":
+            obj_color = Palette.choose_color(d.id)
+        elif color == "by_label":
+            obj_color = Palette.choose_color(d.label)
+        elif color == "random":
+            obj_color = Palette.choose_color(np.random.rand())
+        else:
+            obj_color = parse_color(color)
+
+        if text_color is None:
+            obj_text_color = obj_color
+        else:
+            obj_text_color = color
```

Comment:
Shouldn't we assign `text_color` or its parsed color to `obj_text_color` instead of `color`?